### PR TITLE
ERC20 Support

### PR DESCRIPTION
**Pull request by rajranjan0608**
With this PR, we can add ERC20 faucets as well. See below a sample configuration, for adding Wrapped WAGMI token from the WAGMI subnet (HOSTID)

```json
"erc20tokens": [
        {
            "ID": "WWGMI",
            "HOSTID": "WAGMI",
            "NAME": "Wrapped WAGMI",
            "TOKEN": "WWGMI",
            "IMAGE": "IMAGE_URL"
            "CONTRACTADDRESS": "0x3Ee7094DADda15810F191DD6AcF7E4FFa37571e4",
            "DRIP_AMOUNT": 2000000000,
            "GASLIMIT": "150000",
            "DECIMALS": 18,
            "RATELIMIT": {
                "MAX_LIMIT": 1,
                "WINDOW_SIZE": 1440
            }
        }
    ]
```
It will derive the rest of the information like RPC, MAX_FEE, etc from the HOSTID

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -107,5 +107,35 @@
                 "WINDOW_SIZE": 1440
             }
         }
+    ],
+    "erc20tokens": [
```
Comment (patrick-ogrady) [inline config.json:111]: Add WAVAX on WAGMI: 0x21cf0eB2E3Ab483a67C900b27dA8F34185991982 (https://subnets.avax.network/wagmi/wagmi-chain-testnet/explorer/token/0x21cf0eB2E3Ab483a67C900b27dA8F34185991982)

File: config.json [hunk 2]
```diff
@@ -107,5 +107,35 @@
...
+            "RATELIMIT": {
+                "MAX_LIMIT": 1,
+                "WINDOW_SIZE": 1440
+            }
+        },
+        {
+            "ID": "WWGMI",
+            "HOSTID": "WAGMI",
+            "NAME": "Wrapped WAGMI",
+            "TOKEN": "WWGMI",
```
Comment (patrick-ogrady) [inline config.json]: This should be `WWGM` not `WWGMI`

File: vms/evm.ts [hunk 3]
```diff
@@ -1,199 +1,273 @@
...
         } catch(err: any) {
-            this.error = true;
-            this.log.error(err.message);
+            this.error = true
+            this.log.error(err.message)
         }
     }
 
-    async sendToken(receiver: string, cb: (param: SendTokenResponse) => void): Promise<void> {
+    // Function to issue transfer transaction. For ERC20 transfers, id will be string
```
Comment (patrick-ogrady) [inline vms/evm.ts]: nit: it

Comment (rajranjan0608) [reply vms/evm.ts]: Added more explanation. Actually, it's `id` representing ERC20 token ID as mentioned in the config file.

**Review by patrick-ogrady — APPROVED** [review]

**Pull request merged**